core/server/api_container/server/startosis_engine/runtime_value_store/recipe_result_repository.go

```diff
@@ -59,38 +57,17 @@ func (repository *recipeResultRepository) SaveKey(
 	return nil
 }
 
-// Save store recipe result values into the repository, and it only accepts comparables of
-// starlark.String, starlark.Int and starlark.Bool so far
 func (repository *recipeResultRepository) Save(
 	uuid string,
-	value map[string]starlark.Comparable,
+	value map[string]string,
```

Comment:
would is be possible by any chance to have the repository also handle the serialization/deserialization? I.e. have the repository hold the StarlarkValueSerde object and use it here rather than upstream

Reply:
it's possible, I'll do it